pages/linux/pacdiff.md

```diff
@@ -0,0 +1,28 @@
+# pacdiff
+
+> Pacorig, pacnew and pacsave maintenance utility for pacman.
+> More information: <https://manned.org/pacdiff>.
+
+- Look for files that need maintenance:
+
+`pacdiff`
+
+- Use sudo and sudoedit to remove and merge files:
+
+`pacdiff --sudo`
+
+- Store the original version of any file you `(O)verwrite with` as a `.bak`:
+
+`pacdiff -s --backup`
+
+- Override the program for `(M)erge`ing changes (`vim -d` is default):
+
+`DIFFPROG={{meld}} pacdiff`
+
+- Scan for files with `locate` instead of searching pacmandb to find untracked files:
```

Comment:
I'm not sure what this mean.

Reply:
Sometimes `.pacnew` files end up on your system without being tracked in pacmandb -- I have personally had this happen. Instead of querying the database, the locate flag will scan your filesystem for matching files. How should the wording be clarified?

Reply:
```suggestion
- Scan for configuration files with `locate` instead of using pacman database:
```